CHANGELOG.md

```diff
@@ -1,25 +1,29 @@
 # Changelog
 
+## Unreleased
+
+- Feat: Add support to load release variable from environment(#40)
```

Comment:
Revert and keep the pattern with the `*` prefix.

Reply:
Sorry. Seems It runs my prettier or markdown lint from my vscode. I will update PR soon.